example-deployment/docker/reporting-server.dockerfile

```diff
@@ -1,29 +1,30 @@
-FROM rmf-web/builder
+FROM python:3.8.11-buster
+
+RUN apt-get update && apt-get install -y curl && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && apt-get install -y nodejs python3-pip
+
+RUN pip3 install pipenv
 
 SHELL ["bash", "-c"]
 COPY . /root/rmf-web
-RUN . /opt/rmf/setup.bash && \
-  npm config set unsafe-perm && \
+RUN npm config set unsafe-perm && \
   cd /root/rmf-web && \
   CI=1 npm install -g lerna@4 && lerna bootstrap --scope=reporting-server && \
   cd packages/reporting-server && \
   npm run prepack
 
-FROM rmf-web/builder
+FROM python:3.8.11-buster
+
+RUN apt-get update && apt-get install -y python3-pip
+RUN pip3 install pipenv
 
 COPY --from=0 /root/rmf-web/packages/reporting-server/dist/ .
```

Comment:
I don't think we need this, or we shouldn't have this, `pip3 install` should install whatever is necessary to run `reporting-server`.

Reply:
Yeah, you're right I removed those.